cwl/src/main/scala/cwl/CwlExpressionCommandPart.scala

```diff
@@ -69,8 +69,8 @@ abstract class CommandLineBindingCommandPart(commandLineBinding: CommandLineBind
     }
     
     def processValue(womValue: WomValue): List[String] = womValue match {
-      case WomOptionalValue(_, Some(womValue)) => processValue(valueMapper(womValue))
-      case _: WomString | _: WomInteger | _: WomFile => handlePrefix(valueMapper(womValue).valueString)
+      case WomOptionalValue(_, Some(value)) => processValue(valueMapper(value))
+      case _: WomString | _: WomInteger | _: WomFile | _: WomLong | _: WomFloat => handlePrefix(valueMapper(womValue).valueString)
```

Comment:
long and float inputs were being silently omitted from the command line

Reply:
TOL is there any way to make that non-silent for future omissions of the same sort?

Reply:
good idea